subsys/net/ip/net_shell.c

```diff
@@ -1353,13 +1397,26 @@ int net_shell_cmd_http(int argc, char *argv[])
 		if (strcmp(argv[arg], "monitor") == 0) {
 			printk("Activating HTTP monitor. Type \"net http\" "
 			       "to disable HTTP connection monitoring.\n");
+#if defined(CONFIG_NET_DEBUG_HTTP_CONN) && defined(CONFIG_HTTP_SERVER)
```

Comment:
There's a placeholder function defined (#define http_server_cb(...) NULL) when CONFIG_NET_DEBUG_HTTP_CONN && CONFIG_HTTP_SERVER aren't defined, but we're checking for them here?  Should we remove the placeholder or remove the checks here? (and below)

Reply:
Yes, will fix